types/V1.mli

```diff
@@ -299,38 +302,53 @@ module type BLOCK = sig
 
 end
 
-(** {1 Network stack}
+(** {1 Network stack} *)
+
+module Network : sig
+  type error = [
+    | `Unknown of string (** an undiagnosed error *)
```

Comment:
sure, and imho remove `` `Disconnected`` (see https://lists.xenproject.org/archives/html/mirageos-devel/2016-10/msg00020.html).  In here I intentionally did not rename or remove anything, but `only` put in result and centralised the error polyvar (and pp).